test/system/notify-liquidated.test.js

```diff
@@ -0,0 +1,167 @@
+const { expect } = require("chai")
+const {
+  to1e18,
+  impersonateAccount,
+  resetFork,
+  increaseTime,
+} = require("../helpers/contract-test-helpers")
+const Auction = require("../../artifacts/contracts/Auction.sol/Auction.json")
+
+const describeFn =
+  process.env.NODE_ENV === "system-test" ? describe : describe.skip
+
+// This system test scenario checks the liquidation happy path. It is meant
+// to be executed on Hardhat Network with mainnet forking enabled. At the start,
+// the fork is being reset to the specific starting block which determines the
+// initial test state. This test uses the real tBTC token contract and a
+// deposit (https://allthekeeps.com/deposit/0x55d8b1dd88e60d12c81b5479186c15d07555db9d)
+// which is ready to be liquidated at the starting block. The bidder which
+// takes the offer is also a real account with actual tBTC balance. At the
+// end of the scenario, the risk manager should liquidate the deposit successfully,
+// and 75% of the deposit's bonded amount should land on the signer bonds
+// swap strategy contract.
```

Comment:
I think you forgot to update the test description.

Reply:
Yup, thx, reworded here 5b800bd